neural_modelling/src/neuron/plasticity/stdp/synapse_dynamics_stdp_mad_impl.c

```diff
@@ -118,6 +118,51 @@ bool synapse_dynamics_initialise(
     return true;
 }
 
+//---------------------------------------
```

Comment:
Is there a reason code is moved from 1 place to 2 different places?

Reply:
Yes - the code doesn't actually compile successfully when in the common header (because, I think, the plastic_region_data is not fully defined properly at that point).

Reply:
Also, the code may be duplicated at the moment but I could foresee there being additional information that could be printed here in the neuromodulation implementation.

Reply:
thanks for the info